Cargo.toml

```diff
@@ -4,3 +4,6 @@ members = [
   "stack-graphs",
   "tree-sitter-stack-graphs",
 ]
+
+[patch.crates-io]
+tree-sitter-graph = { path = "../tree-sitter-graph" }
```

Comment:
This looks superfluous — will this go away once tree-sitter/tree-sitter-graph#94 is merged and released in a new version?

Reply:
Yes! This is basically so that I can submodule everything in Aleph and
1. do local development in all dependencies without having to push to remote to get branch dependencies working, and
2. deploy that aleph branch and test all changes together easily.

I marked the commit where I introduced this as temp, but I realize now that I should have added a comment here as well to explain this.

Reply:
I added a comment and replaced the `path` with `git` dependencies so that all projects now build in CI as well.